src/org/labkey/test/tests/SimpleModuleTest.java

```diff
@@ -1069,29 +1070,44 @@ protected void createList()
 
         log("Create list in subfolder to prevent query validation failure");
         createPeopleListInFolder(FOLDER_NAME);
-
+        projectMenu().navigateToFolder(getProjectName(), "subfolder");
         log("Create list in container tab containers to prevent query validation failure");
-        createPeopleListInTab(STUDY_FOLDER_TAB_LABEL);
-        createPeopleListInTab(ASSAY_FOLDER_TAB_LABEL);
+        createPeopleListInTab(STUDY_FOLDER_TAB_LABEL, getProjectName() + "/subfolder/Study Container Tab");
+        createPeopleListInTab(ASSAY_FOLDER_TAB_LABEL, getProjectName() + "/subfolder/Assay Container Tab 2");
     }
 
-    private void createPeopleListInFolder(String folderName)
+    private void createPeopleListInFolder(String folderName) throws Exception
     {
         String containerPath = folderName.equals(getProjectName()) ? getProjectName() : getProjectName() + "/" + folderName;
-        _listHelper.createList(containerPath, LIST_NAME,
-                ListHelper.ListColumnType.AutoInteger, "Key",
-                new ListHelper.ListColumn("Name", "Name", ListHelper.ListColumnType.String, "Name"),
-                new ListHelper.ListColumn("Age", "Age", ListHelper.ListColumnType.Integer, "Age"),
-                new ListHelper.ListColumn("Crazy", "Crazy", ListHelper.ListColumnType.Boolean, "Crazy?"));
+        TestDataGenerator dgen = new TestDataGenerator(new FieldDefinition.LookupInfo(containerPath, "lists", LIST_NAME))
+                .withColumns(List.of(
+                        TestDataGenerator.simpleFieldDef("Name", FieldDefinition.ColumnType.String).setDescription("Name"),
+                        TestDataGenerator.simpleFieldDef("Age", FieldDefinition.ColumnType.Integer).setDescription("Age"),
+                        TestDataGenerator.simpleFieldDef("Crazy", FieldDefinition.ColumnType.Boolean).setDescription("Crazy?")
+                ));
+        dgen.createList(createDefaultConnection(true), "Key");
+        goToManageLists();
+        _listHelper.goToList(LIST_NAME);
     }
 
-    private void createPeopleListInTab(String tabLabel)
+    private void createPeopleListInTab(String tabLabel, String containerPath) throws Exception
```

Comment:
Add a TODO or something to switch this back to using the UI.
This is a fine workaround to get the test working but it isn't testing folder tab functionality.

Reply:
okay